.github/workflows/check_sort_dbt_yaml_files.yaml

```diff
@@ -10,15 +10,22 @@ jobs:
   check-sort-dbt-yaml-files:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v4
-    - name: Set up Python
+    - name: Checkout
+      uses: actions/checkout@v4
+
+    - name: Setup Python
       uses: actions/setup-python@v4
       with:
-        python-version: '3.x'
-    - name: Install dependencies
+        python-version: '3.11'
+        cache: pip
+
+    - name: Install Python dependencies
+      working-directory: dbt
+      shell: bash
       run: |
-        python -m pip install --upgrade pip
-        pip install pyyaml
+        # shellcheck disable=SC2102
```

Comment:
Shellcheck gets mad about the `[]` around the optional deps, so need to disable this rule.

Reply:
**[Suggestion, non-blocking]** Could we avoid disabling the rule if we quoted the `.[ci_checks]` argument, per [the shellcheck docs](https://www.shellcheck.net/wiki/SC2102#exceptions)?

Reply:
Done in 40245bf!